test/lime2.ovpn

```diff
@@ -0,0 +1,110 @@
+
+client
+nobind
+dev tun
+remote-cert-tls server
+
+remote 192.168.1.141 1194 udp
+
+<key>
+-----BEGIN PRIVATE KEY-----

```

Comment:
yo dawg are you really versionning a working private key ? :s

Reply:
local testing key, but yes I need to remove-it from here and have a script for generate the hypercube for testing